Documentation/compatibility/aspnet-472-compat-doc.md

```diff
@@ -0,0 +1,32 @@
+## Appsetting "dataAnnotations:dataTypeAttribute:disableRegEx" is On by Default in .NET 4.7.2
```

Comment:
Better to rephrase this as: 
\`dataAnnotations:dataTypeAttribute:disableRegEx` Appsetting is on by default